threedi_scenario_downloader/downloader.py

```diff
@@ -29,15 +29,22 @@
     "offset": "offset",
 }
 
+"""results endpoint"""
```

Comment:
```suggestion
# results endpoint
```

Commentaar is netter dan "een niet-gebruikte string midden in de code te zetten". Er gaat niets stuk van, maar het is niet netjes :-)

Als er zo'n string direct na een class/functie/methode staat wordt 'ie door python automatisch opgepikt als "docstring". Maar dat is dus niet bedoeld voor commentaar ergens halverwege.